src/omotes_simulator_core/entities/esdl_object.py

```diff
@@ -70,21 +68,17 @@ def get_connected_assets(self, asset_id: str, port: Port) -> List[Tuple[str, Por
         with the type of port they are connected to.
 
         :param str id: id of the asset for which we want to know the connected assets
-        :param Port port: port for which the connected assets need to be returned.
-        :return: List of list which the id of the connected assets and the connected port.
+        :param str port: port for which the connected assets need to be returned.
```

Comment:
Fix parameter name is port_id not port. Kun je ook de 'docstring' iets duidelijker maken?

Reply:
aangepast